src/lib/charm/openstack/keystone_saml_mellon.py

```diff
@@ -256,6 +245,13 @@ class KeystoneSAMLMellonCharm(charms_openstack.charm.OpenStackCharm):
     # First release supported
     release = 'mitaka'
 
+    release_pkg = 'keystone-common'
```

Comment:
I believe this is the default, so it might mot be necessary

Reply:
This is a subordinate to Keystone and therefore needs to be told what package to look at.